book/logistics.md

```diff
@@ -10,11 +10,12 @@
 
 * Tutorials and courses will occur in [Alder Hall](https://goo.gl/maps/GebqB5Gk6hoA3rq1A), with entrance from the ground
   level on the south side.
+* Guests attending via Zoom will be sent a link at 8:00 a.m. (PDT) via the general Slack channel to join at 8:30 a.m. (PDT) each morning
 * Doors open at 8:30 am each morning (Monday-Friday). We will start the morning
   sessions at 9:00 am sharp.
 * Due to space limitations, only registered GeoSMART participants will be allowed
   to attend the tutorials and open working sessions.
-* Participants are expected to attend all scheduled activities. Contact {{ contact_email }}
+* Participants are expected to attend all scheduled activities. Contact [geohack@uw.edu](mailto:geohack@uw.edu)
```

Comment:
```suggestion
* Participants are expected to attend all scheduled activities. Contact {{ contact_email }}
```

(the {{}} fields get filled automatically by the website build script and are set here  https://github.com/geo-smart/2023-hackweek-website/blob/main/book/_config.yml#L12

see: https://geosmart.hackweek.io/logistics.html